script/circleci

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+case $CIRCLE_NODE_INDEX in
+  0 )
+    bundle exec rake test
```

Comment:
> Or... buildkite?

If you give me your credit card to pay for the ec2 instances then sure no problem :p 

More seriously, yes Travis build matrix is definitely better and people are used to it. Also I think Shopify pays for Travis so you should not experience as much queueing than on my personal projects.

@rafaelfranca I'd say let's give a try to travis in this PR. You want to handle it?